activerecord/test/cases/multi_db_migration_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+require "models/person"
+
+# In a multidatabse setup, migrations live in seperate directories.
+# The check for duplicate migration names, does not take this into account.
+# So you __can__ end up having multiple migration with the same name.
+# 
+# Imagine this migration being added to "the other" database
+# in a multidatabase setup.
+#
+# When running `rails db:migrate` from the console, this other migration
+# might be loaded __before__ running the migrations for "this" database.
+#
+# Because they have the same name, we have to ensure that the
+# contents of this migration is not carried over from "the other" migrations.
```

Comment:
We don't usually comment our tests like this, can you remove and make sure it's in the commit message instead? Also I think it'd be better to move this into the migrator_test file rather than make a new one.